jax/_src/compilation_cache.py

```diff
@@ -65,7 +66,28 @@ def set_once_cache_used(f) -> None:
 
 def get_file_cache(path: str) -> tuple[CacheInterface, str] | None:
   """Returns the file cache and the path to the cache."""
-  return GFileCache(path), path
+
+  # `etils[epath]` depends on `fsspec`. If users have `etils[epath]` installed,
```

Comment:
Nit: move into the function or make a docstring.

Reply:
Done